scripts/compare-versions-warning.js

```diff
@@ -0,0 +1,33 @@
+module.exports = function () {
+
+	var fs = require('node:fs');
+	var process = require('process');
+	var cv = require('compare-versions');
+
+	if ((process.mainModule.filename).includes("update-all-files")) {
+		return
+	}
+
+	var package_json = JSON.parse(fs.readFileSync(process.cwd() + '/node_modules/browserslist-config-baseline/package.json', 'utf-8'));
+
+	var warning_message = ` Your locally installed version of browserslist-config-baseline
+ is out of date and your Baseline Widely Available definition may be
+ inaccurate.  Please run \`npm i browserslist-config-baseline@latest\` to
+ get the latest version.`
```

Comment:
```suggestion
var warning_message = (
	"browserslist-config-baseline has been updated. To use the latest " +
	"Widely Available browser’s versions, please run:\n" +
	"\n" +
	"  npm i browserslist-config-baseline@latest"
)
```

It might also make sense to check what package manager the project uses (e.g. [like this](https://github.com/browserslist/update-db/blob/main/index.js#L41-L46)), but that might be an overkill?

Reply:
Not inclined to go too deep here!  I've put messages for bun and yarn in the console.warn: https://github.com/web-platform-dx/browserslist-config-baseline/blob/bf75495549df505a7d5df258cce91bf182b2a52e/scripts/compare-versions-warning.js#L17-L28